src/bin/find.js

```diff
@@ -33,13 +33,15 @@ const getRuleURI = require('eslint-rule-documentation');
 const getRuleFinder = require('../lib/rule-finder');
 const cli = require('../lib/cli-util');
 
+module.exports = (async function() {
```

Comment:
```suggestion
(async function () {
```
bin files shouldn't export anything

Reply:
It's for the test cases. The test cases just import them as modules instead of execute in child processes. E.g.,

https://github.com/MichaelDeBoey/eslint-find-rules/blob/d42b2ec8dac31b692aeab8a49c6293c97fa2dfeb/test/bin/find.js#L66-L71

Would need a lot of changes if we don't export this way. 😬

Reply:
Ah... I don't know any package or manual way that can mock in child processes, and keep the coverage data at the same time. They seem hard. Any idea?

Reply:
I agree, that's difficult. How is this file tested now?

Reply:
it's being tested like normal modules. its test file changes `process.argv` to simulate cli env, and mocks`process.exit` `console.log` and `rule-finder.js` to test the output.

Reply:
oof, that's also gross and hacky.

I'd rather move the logic out of the bin file (in a different PR to the source repo) and make that test cleaner, and then the changes here will be smaller.

Reply:
So we can't run actual child processes. Trying to write explicit `process.exit`s and add their listeners to find out when the cli scripts are finished.

Reply:
I fell into the trap of child processes, thinking another process is the only way to tell when it is finished. Now I'm out. Commit later. 😉

Reply:
> I'd rather move the logic out of the bin file (in a different PR to the source repo) and make that test cleaner, and then the changes here will be smaller.

Yes, I suggest we find or write a tool that wraps these. 👀 That will be much better.

Reply:
What I mean is, take the "bin" file entirely out of the testing equation, and make all the non-bin stuff be tested normally.

Reply:
Oh. Sorry I didn't read it carefully.

Could you finish this for me?

Reply:
I'll give a try later.

Reply:
Done! I must be drunk that everything easy seems so difficult at the moment.

Reply:
Marking as resolved now. Future conversation for this goes to https://github.com/MichaelDeBoey/eslint-find-rules/pull/14.